composer/core/logging/logger_destination.py

```diff
@@ -63,3 +64,40 @@ def log_data(self, state: State, log_level: LogLevel, data: LoggerDataDict):
         """
         del state, log_level, data  # unused
         pass
+
+    def log_file_artifact(
+        self,
+        state: State,
+        log_level: LogLevel,
+        artifact_name: str,
+        file_path: pathlib.Path,
+        *,
+        allow_overwrite: bool,
+    ):
+        """Handle logging of a file artifact stored at ``file_path`` to an artifact named ``artifact_name``.
```

Comment:
Aight I'm starting to get real repetitive now but can we just do a bulk rename of "file artifact" => "artifact"? Or am I dumb and are non-file artifacts a thing

Reply:
Going to leave API as-is for now, so non-file artifacts will not be a future breaking change.